src/modules/LanItems.rb

```diff
@@ -401,7 +402,7 @@ def Import(settings)
       # - creates s390 device eth emulation
       # So, it belongs partly into Import and partly into Write. Note, that
       # the code is currently unable to revert already created emulated device.
-      settings.fetch("s390-devices", {}).each { |rule| createS390Device(rule) } if Arch.s390
+      NetworkAutoYast.instance.activate_s390_devices(settings.fetch("s390-devices", {}))
```

Comment:
I would keep here that s390 condition, so it do not try fancy things if profile contain it accidentally.